tests/core/test_config.py

```diff
@@ -593,6 +593,20 @@ async def test_raw_with_partial_primary_keys(config):
     assert await config.custom("CUSTOM", "primary_key").identifier() is False
 
 
+@pytest.mark.asyncio
+async def test_cast_subclass_default(config):
+    from collections import Counter
```

Comment:
Is there a reason for this import not to be at the top?

Reply:
https://github.com/Cog-Creators/Red-DiscordBot/blob/233278053b906bcbf3fd748b0df89c85fa4ed867/tests/cogs/test_mod.py#L14-L15
I saw imports that are used for a test imported in that test in other places. I assumed since it was used like that elsewhere, that it might need to be like that, but if it should be at the top of the file I can put it there instead.

Reply:
It looks like a code-style thing but we're not consistent about it at all (for comparison, see Downloader, Trivia, or any core tests which put imports at the top). Let's put the import at the top!